asset/css/site-nav.css

```diff
@@ -30,7 +30,7 @@
     -webkit-transition: 0.4s;
 }
 
-#site-nav a:link, #site-nav a:visited, #site-nav a:hover, #site-nav a:active {
+#site-nav a:link, #site-nav a:visited, #site-nav a:active {
```

Comment:
Let's do:
```css
#site-nav a:active,
#site-nav a:link,
#site-nav a:visited {
```

so it's distinct from e.g.:
```css
#site-nav .active .link a:visited {
```

https://stylelint.io/user-guide/rules/selector-list-comma-newline-after/